packages/typo3-docs-theme/assets/sass/_text.scss

```diff
@@ -1,7 +1,25 @@
-
+a {
+    color:inherit;
+    text-underline-offset: 25%;
+    &:hover {
+        text-decoration: none;
```

Comment:
I am not convinced that a hover should remove the underline - I think there may be better ways for signalling hover. Additionally, I miss a :focus for keyboard users.

Reply:
[links-2024-03-04_18.40.09.webm](https://github.com/TYPO3-Documentation/render-guides/assets/2566282/54ca4a8c-c0b3-43fc-b058-bb871f297d6c)

Reply:
How about we change the color of the underline to TYPO3 orange on hover?

Reply:
If that is okay for https://github.com/rfoucard I think that would be good! Let's change the highlight color to orange, and keep the underline, and go from there. We should go an iterative way with constant improvements, else won't get far at all.

To make people happily check out the new rendering, we need people not to be disturbed by little things, which we notice now. Let's do this. :)

Reply:
Hi,
another way can be to change the text-decoration-thickness to 2px, it avoid using only color to give information and it is better than removing the underline.

Reply:
Good idea, I will give it a try!

Reply:
Done